Srigowri_Deogirikar/HTMLAssignment3.html

```diff
@@ -0,0 +1,54 @@
+  <!Doctype html>
+  <html lang="en">
+  <head>
+      <meta charset="UTF-8">
+      <meta http-equiv="X-UA-Compatible" content="IE-edge">
+      <meta name="viewport" content="width=device-width,initial-scale=1.0">
+      <title>Assignment 3</title>
+      
+  </head>
+  <body>
+      <h1>Ambersoft</h1>
+      <ol>
+  <li style="color:rgb(77, 44, 44)">MI-MEAN</li>
+    
```

Comment:
ol tag closing is missing

Reply:
sir its at the last.

Reply:
then only i will get the proper numbering